CPS-????/README.md

```diff
@@ -0,0 +1,173 @@
+---
+CPS: ?
+Title: Intents for Cardano
+Status: Open
+Category: ?
```

Comment:
```suggestion
Category: Tools
```
Since this includes interpretations of new things in the Ledger (not just Ledger changes themselves) and maybe even a language (and therefore tooling) to express these structures (https://github.com/cardano-foundation/CIPs/pull/779#discussion_r1520748340), I would recommend in the next CIP meeting that we apply and/or confirm the `Tools` category.